ts/RailroadMap.ts

```diff
@@ -760,11 +763,25 @@ export class RailroadMap {
 
     private renderBorder(): Element {
         // Border
-        return this.layers.border
-            .rect(4_000_00, 4_000_00)
-            .translate(-2_000_00, -2_000_00)
-            .radius(100_00)
-            .addClass('map-border');
+        if (this.railroad.settings.gameLevelName === 'LakeValley') {
+            return this.layers.border
+                .rect(4_000_00, 4_000_00)
+                .translate(-2_000_00, -2_000_00)
+                .radius(100_00)
+                .addClass('map-border');
+        } else if (this.railroad.settings.gameLevelName === 'AuroraFalls') {
```

Comment:
Can you please share an AF save file for testing?

Reply:
here it is
[!AF_main.zip](https://github.com/scottanderson/railroad.studio/files/14379576/AF_main.zip)

Reply:
it looks like this tho
![image](https://github.com/scottanderson/railroad.studio/assets/126168696/cc9f62f5-fcea-4e80-a628-c7b808205ca6)
![image](https://github.com/scottanderson/railroad.studio/assets/126168696/d5b52ef0-436b-4a41-839f-4a8b785825fb)